index.html

```diff
@@ -0,0 +1,350 @@
+        <a href="" class="desk-see-project-1c">See Project</a>
+      </div>
+    </div>
+
+    <div class="card456">
+      <div class="desk-card-d1 all-cards">
+       <div class="no-hover">
+        <h2 class="desk-prof-art-dash">Profesional Art Printing Data More</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+       </div>
+       <a href="" class="desk-see-project-1c">See Project</a>
+      </div>
+
+      <div class="desk-card-d2 all-cards">
+        <div class="no-hover">
+          <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+        </div>
+        <a href="" class="desk-see-project-1c">See Project</a>
+      </div>
+
+      <div class="desk-card-w1 all-cards">
+       <div class="no-hover">
+        <h2 class="desk-prof-art-w">Website Protfolio </h2>
+        <p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-button-d button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+       </div>
+        <a href="" class="desk-see-project-1c">See Project</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="section-3">
+    <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+        website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+    </div>
+    <hr class="line-3">
+    <div class="about-me-desk">
+      <div class="story-abt-me">
+        <h1 class="h31-desk">About me</h1>
+        <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+          website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.</p>
+        <a class="get-myresume button" href="">Get My Resume</a>
+      </div>
+    </div>
+
+    <ul class="lists">
+      <li class="Lan">Language
```

Comment:
[OPTIONAL]
- [ ] The technologies section of your portfolio page is not correctly centered in the viewport as can be seen from the figma template shown below please kindly adopt your figma design to look closely like the portfolio template you chose.
- figma design
![centered-f](https://user-images.githubusercontent.com/89055502/236587782-3a67ff64-22c7-41f4-a367-8ec2b2ba8b35.png)
- your design
![cent-s](https://user-images.githubusercontent.com/89055502/236587791-8ee9e21d-be04-4dc4-b04b-70fb9372faf5.png)